setup.py

```diff
@@ -0,0 +1,40 @@
+import sys
+
+import setuptools
+
+
+requires = [
+    'base58',
+    'binary-helpers',
+    'coincurve'
+]
+
+tests_require = [
+    'flake8>=3.5.0',
+    'flake8-import-order>=0.17.1',
+    'flake8-print>=3.1.0',
+    'flake8-quotes>=1.0.0',
+    'pytest>=3.6.1',
+    'pytest-cov>=2.5.1'
+]
+
+extras_require = {
+    'test': tests_require,
+    'dev': requires + tests_require
+}
+
+setup_requires = ['pytest-runner'] if {'pytest', 'test', 'ptr'}.intersection(sys.argv) else []
+
+setuptools.setup(
+    name='ark-crypto',
```

Comment:
Will be release as `arkecosystem-crypto` or are scoped packages possible in PHP or with npm?